test-harness/specs/security-scheme-validation-AC-12.only_oas3.txt

```diff
@@ -0,0 +1,39 @@
+====test====
+Adding a required cookie with the name of SESSIONID results in granted access.
+====spec====
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              example: test
+components:
+  schemas:
+    LoginRequest:
```

Comment:
Why is this necessary? Looks like you're not using it, correct?

Reply:
right! thanks, this is a leftover (when I had a login endpoint), will remove